File: POO/temas/6/rb/6.rb
class Frigo
    def initialize
        @color = "blanco"
        @puerta = "cerrada"
        @temperatura = 20
    end

    def abre
        @puerta = "abierta"
    end

    def cierra
        @puerta = "cerrada"
    end

    def fija_temperatura
        puts "se cambia la temperatura"
    end

    def muestra_estado
        puts "Color: #{@color} | Puerta #{@puerta} | #{@temperatura}ºC"
    end
end


mi_frigo = Frigo.new
mi_frigo.muestra_estado
mi_frigo.abre
mi_frigo.muestra_estado
mi_frigo.cierra
mi_frigo.muestra_estado

File: POO/temas/6/py/6.py
class Frigo:
    def __init__(self):
        self.__color = "blanco"
        self.__puerta = "cerrada"
        self.__temperatura = 20

    def abre(self):
        print( "se está abriendo" )

    def cierra(self):
        print( "se está cerrando" )

    def fija_temperatura(self):
        print( "se cambia la temperatura" )

    def muestra_estado(self):
        print( f'Color: {self.__color} | Puerta {self.__puerta} | {self.__temperatura}ºC' )


mi_frigo = Frigo()
mi_frigo.muestra_estado()
